### PR TITLE
introduce new SpotBugs Gradle Plugin

### PR DESCRIPTION
**Pull request by KengoTODA**
refs https://github.com/spotbugs/spotbugs-gradle-plugin/issues/180

### CODE REVIEW

**Comment by KengoTODA** [comment]
`tasks.withType(SpotBugsTask) { ... }` does not work, I will try to find the reason and fix it.

File: build.gradle [hunk 1]
```diff
@@ -1,6 +1,7 @@
 plugins {
   id 'org.sonarqube' version '2.7.1'
   id "com.diffplug.gradle.spotless" version "3.27.1"
+  id "jp.skypencil.spotbugs.snom" version "0.4.13"
```
Comment (ThrawnCA) [inline build.gradle]: What is this?

Comment (KengoTODA) [reply build.gradle]: A gradle plugin implemented from scratch. Will be official spotbugs-gradle-plugin 4.0.0.
refs https://github.com/spotbugs/spotbugs-gradle-plugin/issues/180

**Comment by KengoTODA** [comment]
I'll wait for 4.0.1 release that fixes https://github.com/spotbugs/spotbugs-gradle-plugin/issues/196

**Pull request merged**